Doc/library/os.rst

```diff
@@ -2861,70 +2918,111 @@ features:
         <https://msdn.microsoft.com/en-us/library/aa363788>`_ on
         Windows
```

Comment:
This looks incomplete in the source here, i assume this renders to something different?

Reply:
There's one more line that's not visible in the diff, because I didn't touch these lines. You can see it at https://docs.python.org/3/library/os.html#os.stat_result.st_ino where I think it looks fine